Fix a bug in the "-svd 0" configuration flag that prevented it from
being able to create the "out" directory on 10.6 and earlier.
https://gist.github.com/ryandesign/a7fdea794e1b84a01f15536fabe7f9ff
--- src/OSGLUCCO.m.orig	2018-07-06 23:28:04.000000000 -0500
+++ src/OSGLUCCO.m	2018-07-06 23:31:33.000000000 -0500
@@ -4014,7 +4014,7 @@
 			}
 		} else {
 			if ([fm respondsToSelector:@selector(
-createDirectoryAtURL:withIntermediateDirectories:attributes:error:
+createDirectoryAtPath:withIntermediateDirectories:attributes:error:
 				)])
 			{
 				if ([fm
@@ -4028,7 +4028,7 @@
 				}
 			} else
 			if ([fm respondsToSelector:
-				@selector(createDirectoryAtURL:attributes:)])
+				@selector(createDirectoryAtPath:attributes:)])
 			{
 				if ([fm
 					createDirectoryAtPath:r
